scene/2d/remote_transform_2d.cpp

```diff
@@ -63,7 +63,21 @@ void RemoteTransform2D::_update_remote() {
 		return;
 
 	//todo make faster
-	n->set_global_transform(get_global_transform());
+	if (use_global_coordinates) {
+		if (update_remote_position)
+			n->set_global_position(get_global_position());
+		if (update_remote_rotation)
+			n->set_global_rotation(get_global_rotation());
+		if (update_remote_scale)
+			n->set_global_scale(get_global_scale() + Vector2(1, 1));
```

Comment:
Additionally, if only rotation and scale should be copied, you can still pull this off, with something like:
```cpp
if (update_remote_rotation && update_remote_scale) {
	Transfrom2D global_transform = get_global_transform();
	if (!update_remote_position)
		global_transform.origin = Vector2();
	n->set_global_transform(global_transform);
} else {
	… // Current code goes here
}
```